### ISSUE TITLE
Text color white in Template Library SearchBar.

### ISSUE THREAD

**Issue by arteevraina** [accordproject/web-components#72]
**Describe the bug**
The text color is white inside the Template Library Search bar.

**To Reproduce**
1. Go to [demo](https://cicero-ui.netlify.app/?path=/story/components-template-library--template-library)
2. Click on Template Library Component.
3. Type in search bar.

**Expected behavior**
The text color should be black so that it clearly differentiates it from the background color.

**Screenshots**
![image](https://user-images.githubusercontent.com/43968121/79606883-e6821a00-810f-11ea-93d2-d3709ebbf30b.png)

**Additional context**
I would like to work on this.

**Comment by irmerk** [comment]
This has been fixed.
